db/migrations/02_add_date_to_peeps.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE peeps ADD COLUMN date DATE;
```

Comment:
Separate migration file -- great. And DATE is an appropriate datatype to use. Another option would have been to use TIMESTAMP. That would give you flexibility in the future if it ever becomes necessary to show time information too.